### ISSUE TITLE
[a11y] 11.2.4.7 Focus visible

### ISSUE THREAD

**Issue by tschiebel** [owncloud/android#4369]
audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf
https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de

Note: If the focus is highlighted exclusively by a color change,
the contrast between the focused and unfocused state should be sufficient.
It is recommended that the contrast ratio is at least 3:1.

### Personal Space

- [ ] All interactive elements on the page have a poorly recognizable keyboard focus.
keyboard focus. The contrast between the focused and unfocused state
is well below the recommended contrast ratio of 3:1.
the contrast of the labels on the buttons deteriorates when focusing.
buttons deteriorate.
![image](https://github.com/owncloud/android/assets/132444655/20fb5283-33bc-4899-9c92-b044f7735036)
![image](https://github.com/owncloud/android/assets/132444655/95fff6ba-a34d-4afd-a79e-ed3431ee3cd7)

### Menu

- [ ] All interactive elements on the page have a poorly recognizable keyboard focus. 
keyboard focus. The contrast between the focused and unfocused state 
is well below the recommended contrast ratio of 3:1.
![image](https://github.com/owncloud/android/assets/132444655/0a6d8257-3fd1-4c39-8a6d-b256a678b1c1)

### Share

- [ ] All interactive elements on the page have a poorly recognizable keyboard focus. 
keyboard focus. The contrast between the focused and unfocused state is well below the recommended contrast ratio of 3:1. 
is well below the recommended contrast ratio of 3:1. 
the contrast of the buttons themselves deteriorates when focusing.
![image](https://github.com/owncloud/android/assets/132444655/bdff042a-a10c-4a44-a1d6-bcd7bc383ec0)

### Edit shared Link

- [ ] All interactive elements on the page, with the exception of the input field, are given a 
a poorly recognizable keyboard focus.
![image](https://github.com/owncloud/android/assets/132444655/e8bd8cec-8eba-48e8-967c-d4436b983d07)

### Spaces

- [ ] All interactive elements on the page have a poorly recognizable keyboard focus. 
keyboard focus. The contrast between the focused and unfocused state 
is well below the recommended contrast ratio of 3:1.
e.g:
![image](https://github.com/owncloud/android/assets/132444655/e88c4b30-1192-40a6-9173-8a17a2759867)

### Manage account

- [ ] All interactive elements on the page have a poorly recognizable keyboard focus. 
keyboard focus. The contrast between the focused and unfocused state 
is well below the recommended contrast ratio of 3:1.
e.g: 
![image](https://github.com/owncloud/android/assets/132444655/e9e2190a-c245-46c6-9bbb-a3efda28278d)

**Comment by jesmrec** [comment]
Check if color is set from OS, if it is setup-able. In every case, make a decision and document it.
